pixelish/usr/share/asteroid-launcher/watchfaces/pixelish.qml

```diff
@@ -0,0 +1,160 @@
+/*
+ * Copyright (C) 2021 - Timo Könnecke <github.com/eLtMosen>
+ * Copyright (C) 2022 - Commenter25 <github.com/Commenter25>
+ *
+ * All rights reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation, either version 2.1 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+/* based off digital-shifted */
+
+import QtQuick 2.9
+import QtGraphicalEffects 1.15
+
+Item {
+	id: root
+	
+	property real arcEnd: wallClock.time.getSeconds() * 6
+	onArcEndChanged: canvas.requestPaint()
+
+	/* Uncomment this block for smooth seconds 
+	TODO: fix animating looping back to 0 */
+	/*
+	Behavior on arcEnd {
+			id: animationArcEnd
+			enabled: true
+			NumberAnimation {
+					duration: 1000
+					easing.type: Easing.Linear
+			}
+	}
+	*/
+	
+
+	Canvas {
+			id: canvas
+			visible: !displayAmbient
+			anchors.fill: parent
+			rotation: -90
+			onPaint: {
+					var ctx = getContext("2d")
+					var x = root.width / 2
+					var y = root.height / 2
+					var start = 0
+					var end = Math.PI * (parent.arcEnd / 180)
+					ctx.reset()
+					ctx.beginPath()
+					ctx.lineCap="round"
+					ctx.arc(x, y, (root.width / 2) - parent.height*0.124 / 2, start, end, false)
+					ctx.lineWidth = parent.height * 0.03
+					ctx.strokeStyle = "#CCC"
+					ctx.stroke()
+			}
+			layer.enabled: true
+	}
+
+
+	Text {
+			id: colon
+
+			renderType: Text.NativeRendering
+			horizontalAlignment: Text.AlignHCenter
+			color: "#ffffff"
+			font {
+				pixelSize: root.height * .28
+				family: "Open Sans"
+			}
+
```

Comment:
There is a slight inconsistency in all following `Text` items. In some cases you put an empty line after the property blocks (font/anchors), in some cases not. Not sure what looks best, i tend to not have empty lines between property blocks since they are nicely distinguishable as such. Fort he single text line however, i do an empty line upfront to make it better findable in all Text items.

Reply:
One day I'll learn to be consistent with whitespace... especially need to learn that in CSS :p I tend to just do whatever feels right in my head at the time to separate things.

Reply:
I believe I did what you were saying, though I feel I might've misunderstood, so let me know if it's still unusual